### ISSUE TITLE
Log warning Podnapisi provider

### ISSUE THREAD

**Issue by vincenzogianfelice** [morpheus65535/bazarr#2531]
**Describe the bug**
Every search with podnapisi: ```HTTPSConnectionPool(host='www.podnapisi.net', port=443): Read timed out. (read timeout=30), retrying in 5 seconds...```

**To Reproduce**
1. Go to Systems/Logs

**Software (please complete the following information):**
 - Bazarr: v**1.4.3**

**Comment by zappy16** [comment]
Yeah it's been really bad lately. I don't think there is much we can do from Bazarr side as even loading https://www.podnapisi.net/ in browser can take up to 10 seconds. And even refreshing a few times the site doesn't always load fully or I get different results. See attached for loading the site in two different tabs, I get different renderings. Not sure what is going on on their hosting, but seems to be their end. 

![Screenshot 2024-06-11 at 11 13 49 AM](https://github.com/morpheus65535/bazarr/assets/17417763/b6e30e03-be2d-4788-af61-d26d11d81218)

![Screenshot 2024-06-11 at 11 22 20 AM](https://github.com/morpheus65535/bazarr/assets/17417763/58807fdc-0e80-4e74-9bec-8d7237f80101)

**Comment by zappy16** [comment]
Now I'm just flat out getting "An Error has occurred" when attempting to load their site. I've disabled it on my end as I don't get much from them anyways...

![Screenshot 2024-06-11 at 11 29 12 AM](https://github.com/morpheus65535/bazarr/assets/17417763/70523518-a5f3-42d1-a666-dc82332bf406)

**Comment by morpheus65535** [comment]
I agree, there's not much we can do from our side. The site is in a strange state and this issue has been raised about 65k times by more than 5k users in the last five days.

I'll tag this as broken provider and we'll see how it evolves over the next weeks.

**Comment by morpheus65535** [comment]
> @morpheus65535  it's a shame.  It is one of the few that indexes many subtitles in Italian.  I hope at this point that you will implement https://www.italiansubs.net as soon as possible, I have also opened a ticket for this.  I hope you will consider it.  Unfortunately Italian is never considered!!  😂

I don't prioritize by language but by number of upvotes for a specific feature.

**Comment by gianfelicevincenzo** [comment]
It's been almost two years, and unfortunately, their website is still having problems. I just wanted to let you know...

<img width="823" height="279" alt="Image" src="https://github.com/user-attachments/assets/2fd46942-aa13-4f6d-a198-be78be5bf834" />

**Comment by rumplin** [comment]
can we reduce the amount of requests going to podnapisi.net? I think we are part of the problem that the site can't keep up:

> 2026-01-21 01:02:12|WARNING |retry.api                       |HTTPSConnectionPool(host='podnapisi.net', port=443): Read timed out. (read timeout=30), retrying in 5 seconds...|
> 2026-01-21 01:02:21|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:04:53|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:08:17|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:08:58|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:09:29|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:12:07|WARNING |retry.api                       |HTTPSConnectionPool(host='podnapisi.net', port=443): Max retries exceeded with url: /subtitles/search/old?sXML=1&sL=sl&sK=Hedda&sY=2025 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2eda1f5950>: Failed to establish a new connection: [Errno 101] Network unreachable')), retrying in 5 seconds...|
> 2026-01-21 01:14:26|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:14:46|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:15:00|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:15:55|WARNING |retry.api                       |HTTPSConnectionPool(host='www.podnapisi.net', port=443): Read timed out. (read timeout=30), retrying in 5 seconds...|
> 2026-01-21 01:16:14|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:18:05|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:19:12|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:19:37|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|
> 2026-01-21 01:19:43|WARNING |retry.api                       |('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), retrying in 5 seconds...|

**Comment by morpheus65535** [comment]
@rumplin I've made some changes that may help reduce the load on their side. I hope it will be enough to make this provider usable again. Thanks for the suggestion!

**Comment by Gameplayer13** [comment]
Hey @morpheus65535
The provider is still problematic sadly :( 

> 2026-03-13 02:09:05|INFO    |root                            |Throttling podnapisi for 10 minutes, until 26/03/13 02:19, because of: ConnectionError. Exception info: "'HTTPSConnectionPool(host='www.podnapisi.net', port=443): Max retries exceeded w' ~ podnapisi.py@207"|
2026-03-13 02:09:37|INFO    |root                            |Throttling opensubtitlescom for 6 hours, until 26/03/13 08:09, because of: DownloadLimitExceeded. Exception info: "'Daily download limit reached. 20 subtitles have been downloaded and 0 re' ~ opensubtitlescom.py@568"|
2026-03-13 02:09:53|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|
2026-03-13 02:11:26|INFO    |root                            |BAZARR Finished searching for missing Series Subtitles. Check History for more information.|
2026-03-13 02:24:58|INFO    |root                            |Using podnapisi again after 10 minutes, (disabled because: ConnectionError)|
2026-03-13 02:25:59|INFO    |root                            |Throttling podnapisi for 10 minutes, until 26/03/13 02:35, because of: ConnectionError. Exception info: "'HTTPSConnectionPool(host='podnapisi.net', port=443): Max retries exceeded with ' ~ podnapisi.py@207"|
2026-03-13 02:25:59|INFO    |root                            |Throttling podnapisi for 10 minutes, until 26/03/13 02:35, because of: ConnectionError. Exception info: "'HTTPSConnectionPool(host='podnapisi.net', port=443): Max retries exceeded with ' ~ podnapisi.py@207"|
2026-03-13 02:26:03|INFO    |root                            |Throttling podnapisi for 10 minutes, until 26/03/13 02:36, because of: ConnectionError. Exception info: "'HTTPSConnectionPool(host='www.podnapisi.net', port=443): Max retries exceeded w' ~ podnapisi.py@207"|
2026-03-13 03:40:13|INFO    |root                            |Using podnapisi again after 10 minutes, (disabled because: ConnectionError)|
2026-03-13 04:00:02|INFO    |root                            |BAZARR All existing movie subtitles indexed from disk.|
2026-03-13 04:00:17|INFO    |root                            |BAZARR All existing episode subtitles indexed from disk.|
2026-03-13 05:20:44|INFO    |root                            |Throttling podnapisi for 1 hour, until 26/03/13 06:20, because of: ReadTimeout. Exception info: "'HTTPSConnectionPool(host='www.podnapisi.net', port=443): Read timed out. (read ' ~ podnapisi.py@207"|
2026-03-13 05:20:56|INFO    |root                            |BAZARR Finished searching for missing Movies Subtitles. Check History for more information.|

**Comment by morpheus65535** [comment]
@Gameplayer13 There's not much that can be done unfortunately. They've implemented latest cloudflare protection and the Python module (cloudscraper) we're using to circumvent Cloudflare protection still doesn't properly deals with this properly. We're ready to upgrade this module when it's going to be supported.

**Comment by morpheus65535** [comment]
Podnapisi is officially closed since March 25th 2026.
